packages/common-test-utils/src/index.ts

```diff
@@ -810,3 +810,104 @@ export class NoteTestPresetsV2 {
     ];
   }
 }
+
+/**
+ * Utility class to make the tests against vault to be concise while having descriptive
+ * failure messages. */
+export class VaultAsserter {
```

Comment:
I think these utility functions could be part of `VaultUtils`. Or if you want to isolate utilities strictly for testing, I suggest we keep the naming convention consistent with the others and call it say, `VaultTestUtils`.

Reply:
Renamed to VaultTestUtils

Reply:
removed